# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.13/noctua_fan_mod_always_run_full_speed.diff | git apply -v
#
# This patch set fan speed to max (65535), suitable for EONs with Noctua fan mode
# courtesy of Erich Moraga (https://github.com/ErichMoraga/)
#
===================================================================
--- a/selfdrive/thermald.py
+++ b/selfdrive/thermald.py
@@ -80,9 +80,9 @@
 # temp thresholds to control fan speed - low hysteresis
 _TEMP_THRS_L = [42.5, 57.5, 72.5, 10000]
 # fan speed options
-_FAN_SPEEDS = [0, 16384, 32768, 65535]
+_FAN_SPEEDS = [65535, 65535, 65535, 65535]
 # max fan speed only allowed if battery is hot
-_BAT_TEMP_THERSHOLD = 45.
+_BAT_TEMP_THERSHOLD = 20.
 
 
 def handle_fan(max_cpu_temp, bat_temp, fan_speed):
